README.md

```diff
@@ -29,284 +30,57 @@ pip install graphdatascience
-Some examples are (where `pipe` is a machine learning training pipeline object):
-
-```python
-res = gds.beta.pipeline.list()
-assert len(res) == 1  # Exactly one pipeline is in the catalog
-
-res = gds.beta.pipeline.drop(pipe)
-assert res["pipelineName"] == pipe.name()
-```
-
-Further, there's a call named `gds.pipeline.get` (`graphdatascience` only).
-It takes a pipeline name as input and returns a training pipeline object, if a pipeline of that name exists in the user's pipeline catalog.
-The idea is to have a way of creating pipeline objects for already existing pipelines, without having to create them again.
-
-
-### Model catalog utils
-
-All procedures from the [GDS Model catalog](https://neo4j.com/docs/graph-data-science/current/model-catalog/) are supported with `graphdatascience`.
-Some examples are (where `model` is a machine learning model object):
-
-```python
-res = gds.beta.model.list()
-assert len(res) == 1  # Exactly one model is loaded
-
-res = gds.beta.model.drop(model)
-assert res["modelInfo"]["modelName"] == model.name()
-```
-
-Further, there's a call named `gds.model.get` (`graphdatascience` only).
-It takes a model name as input and returns a model object, if a model of that name exists in the user's model catalog.
-The idea is to have a way of creating model objects for already loaded models, without having to create them again.
-
-
-### Node matching without Cypher
-
-When calling path finding or topological link prediction algorithms one has to provide specific nodes as input arguments.
-When using the GDS procedure API directly to call such algorithms, typically Cypher `MATCH` statements are used in order to find valid representations of input nodes of interest, see eg. [this example in the GDS docs](https://neo4j.com/docs/graph-data-science/current/algorithms/dijkstra-source-target/#algorithms-dijkstra-source-target-examples-stream).
-To simplify this, `graphdatascience` provides a utility function, `gds.find_node_id`, for letting one find nodes without using Cypher.
-
-Below is an example of how this can be done (supposing `G` is a projected `Graph` with `City` nodes having `name` properties):
-
-```python
-# gds.find_node_id takes a list of labels and a dictionary of
-# property key-value pairs
-source_id = gds.find_node_id(["City"], {"name": "New York"})
-target_id = gds.find_node_id(["City"], {"name": "Philadelphia"})
-
-res = gds.shortestPath.dijkstra.stream(G, sourceNode=source_id, targetNode=target_id)
-assert res["totalCost"][0] == 100
-```
-
-The nodes found by `gds.find_node_id` are those that have all labels specified and fully match all property key-value pairs given.
-Note that exactly one node per method call must be matched.
-
-For more advanced filtering we recommend users do matching via Cypher's `MATCH`.
+* [Product recommendations with kNN based on FastRP embeddings](https://github.com/neo4j/graph-data-science-client/tree/main/examples/fastrp-and-knn.ipynb)
```

Comment:
Noticing that this list is rather incomplete (not related to this PR)

Reply:
Ach, I should've double-checked it. I'll have a look and cross-reference.

Reply:
Fixed in this PR https://github.com/neo4j/graph-data-science-client/pull/249

Reply:
done